authentication/models.py

```diff
@@ -146,6 +142,17 @@ def _generate_jwt_token(self):
         return token.decode('utf-8')
 
 
+class CardInfo(BaseAbstractModel):
+    embed_token = models.CharField(unique=True, max_length=300)
+    card_number = models.CharField(unique=True, max_length=20)
+    card_expiry = models.CharField(max_length=6)
+    card_brand = models.CharField(max_length=100)
+    user = models.ForeignKey(User, null=True, on_delete=models.CASCADE)
```

Comment:
@Niyitangasam  Making a **ForeignKey** field `nullable` is not a good database practice.

Reply:
@Victoradukwu, I know that it's not good practice but as in my implementation I had first to save card info and then update it with the user, If I didn't allow it to be the first null it wouldn't work but I always make sure that the foreign key is saved.

Reply:
@Niyitangasam Let's sync on this by 10 am CAT on Monday.